daemon/graphdriver/lcow/remotefs_file.go

```diff
@@ -33,7 +33,7 @@ func (l *lcowfs) OpenFile(path string, flag int, perm os.FileMode) (_ driver.Fil
 	flagStr := strconv.FormatInt(int64(flag), 10)
 	permStr := strconv.FormatUint(uint64(perm), 8)
 
-	commandLine := fmt.Sprintf("%s %s %s %s", remotefs.RemotefsCmd, remotefs.OpenFileCmd, flagStr, permStr)
+	commandLine := fmt.Sprintf("%s %s %s %s %s", remotefs.RemotefsCmd, remotefs.OpenFileCmd, path, flagStr, permStr)
```

Comment:
Should we have validation of the arguments that were passed? (`path` not empty, `flag` not zero? (not sure what `flag` is without description), `permStr` being a valid value?

Also; perhaps we should change the `fmt.Errorf("failed to open file pipes %s: %s", path, err)` to use `%q` for the path, then it becomes more aparent if the value is empty

Reply:
Defers largely to @gupta-ak, but the remote command will fail when executed with invalid parameters

Reply:
Yeah, the remote command on the GCS side validates the parameters, so it will fail there.